ui/pages/onboarding-flow/creation-successful/creation-successful.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import reactRouterDom from 'react-router-dom';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
```

Comment:
Are we using this file or are we using `test/jest/rendering`?

Reply:
I think you're right we are moving to use of `test/jest/rendering`. Will swap this out.